test/animationinfos_test.cpp

```diff
@@ -1,7 +1,7 @@
 #include <gtest/gtest.h>
 
 #include "nthread.h"
-#include "player.h"
+#include "animationinfos.h"
```

Comment:
This looks a bit hard to read for me... should we consider naming it `animation_info` instead?

Reply:
I suggest we create an engine folder and put it there, then just call it `engine/animation.h`. Then as we refactor `engine.cpp/h`, `scrollrt.cpp/h` and `engine.cpp/h` we can split it up into other files in that folder.

Reply:
I moved it to new engine-folder but stayed with `AnimationInfo` cause
- It doesn't contain/own the Animation-Metadata (Gfx, Number-of-Frames for all Animations) so I would like to distinguish both.
- I would like to name the file identically to the containing class.

But if you want I can rename only the file or both. Just say so.

Reply:
> It doesn't contain/own the Animation-Metadata (Gfx, Number-of-Frames for all Animations) so I would like to distinguish both.
> I would like to name the file identically to the containing class.

The relationship between header and class in C++ is not necessarily 1 to 1. Usually, you have a header that groups a certain number of elements related to the same "area". In that sense, naming the header just `animation.h` (or `animation.hpp`... whatever is the current standard) is acceptable to me.

I'd totally agree with you if this was in another context, like C#, where it is a best practice to have one file per class.